app/src/main/java/org/mozilla/fenix/home/sessioncontrol/viewholders/pocket/PocketStoriesComposables.kt

```diff
@@ -169,19 +179,174 @@ fun PocketStories(
                 Column(
                     Modifier.padding(end = if (index == halfStoriesIndex) 0.dp else 8.dp)
                 ) {
-                    PocketStory(item, client)
+                    key(item.title) {
+                        PocketStory(item, client)
+                    }
```

Comment:
Why do we use `key()` here? Is it to avoid recomposition of `PocketStory` unless the title changes? Why the title only though? 🤔

Reply:
The title seemed like a small thing to check and enough to protect against recompositions. The stories are static in nature with no property changing. 
Tested a bit now, don't see any recomposition. Same calls with or without `key`. Will remove it.